digital-twin-model/dtdl/v3/content/hvac.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
Is this the same as the HVAC DTDL used in other samples? If so, can we consolidate them so that there's less duplication?

Reply:
All the samples use the same DTDL.  There is no duplication.